src/components/Listing/Listing.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Listing from './Listing';
+
+describe('<Listing />', () => {
+  const output = shallow(
+    <Listing>
+      <li>List item 1</li>
+      <li>List item 2</li>
+      <li>List item 3</li>
+    </Listing>
```

Comment:
Small thing - but is this rendering 3 <li>s as children of a <div>?

Reply:
Ha, good spot. The component was set to `ul` as  default originally hence the list items